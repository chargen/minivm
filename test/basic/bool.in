t = true
f = false
print t
print f
print t + t
print t * f
print f / t
print t / (f / t + t)
print t * 3.14 + f / 3.14
print 3.14 / (t + f)
