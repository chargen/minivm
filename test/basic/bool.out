true
false
2
0
0
1
3.140000000
3.140000000
true
true
true
false
false
false
